docs/pages/installation.mdx

```diff
@@ -360,12 +360,12 @@ script](#one-line-installation-script) or manually install Teleport from a
    install binaries:
 
    ```code
-   $ curl https://get.gravitational.com/${TELEPORT_PKG?}-v(=teleport.version=)-linux-${SYSTEM_ARCH?}-bin.tar.gz.sha256
+   $ curl https://cdn.teleport.dev/${TELEPORT_PKG?}-v${TELEPORT_VERSION}-linux-${SYSTEM_ARCH?}-bin.tar.gz.sha256
```

Comment:
Not familiar enough with the docs code to say, but was the change in how variables are referenced intentional? From `=teleport.version=` to `TELEPORT_VERSION`.

Reply:
The earlier instructions say to set the `TELEPORT_VERSION` here. But then then that has no impact in running if you didn't switch out the version the docs inserts. The earlier instructions will automatically put in a initial value for you of the most recent version.

 Specify your system Teleport version and system architecture by assigning an
   environment variable:
   ```code
   $ TELEPORT_VERSION=(=teleport.version=)
   $ SYSTEM_ARCH=""
```